mindbender/plugins/collect_project_editorial_info.py

```diff
@@ -0,0 +1,36 @@
+# collect_project_editorial_info.py
+import pyblish.api
+
+
+class CollectMindbenderEditInfo(pyblish.api.ContextPlugin):
+    """Store projects editorial info at the time of publish"""
+
+    label = "Collect Project Edit Info"
+    order = pyblish.api.CollectorOrder
+    hosts = ["maya"]
+    families = [
+        # More will come
+        "mindbender.animation"
+    ]
+
+    def process(self, context):
+        import os
+
+        mindbender_fps = os.getenv("MINDBENDER_FPS")
+        mindbender_edit_in = os.getenv("MINDBENDER_EDIT_IN")
+        mindbender_edit_out = os.getenv("MINDBENDER_EDIT_OUT")
+
+        if mindbender_fps == "" or mindbender_fps is None:
+            context.data["project_fps"] = 25
```

Comment:
Pyblish naming convention for data is `mixedCase`.

- https://api.pyblish.com/pages/AbstractEntity.data.html